include/parlay/scheduler.h

```diff
@@ -39,10 +40,20 @@
 
 #include "internal/work_stealing_job.h"
 
+#ifdef PARLAY_INTERACTIVE
+#include "internal/atomic_wait.h"
+#endif
+
 // IWYU pragma: no_include <bits/this_thread_sleep.h>
 
 namespace parlay {
 
+#if __has_cpp_attribute(unlikely)
```

Comment:
There's a file, `include/parlay/portability.h`, where these kinds of macros should go.